### ISSUE TITLE
Debug prints

### ISSUE THREAD

**Issue by lucaTagliapietra** [robotology/robotology-superbuild#18]
While configuring the superbuild, I see this prints:
~~~
YCM: _dep_component_set = 1, _dep_component = core
RTF: _dep_component_set = 1, _dep_component = core
YARP: _dep_component_set = 1, _dep_component = core
YARP: _dep_component_set = 1, _dep_component = core
ICUB: _dep_component_set = 1, _dep_component = iCub
YARP: _dep_component_set = 1, _dep_component = core
ICUB: _dep_component_set = 1, _dep_component = iCub
iDynTree: _dep_component_set = 1, _dep_component = dynamics
wholeBodyInterface: _dep_component_set = 1, _dep_component = dynamics
YARP: _dep_component_set = 1, _dep_component = core
ICUB: _dep_component_set = 1, _dep_component = iCub
iDynTree: _dep_component_set = 1, _dep_component = dynamics
wholeBodyInterface: _dep_component_set = 1, _dep_component = dynamics
yarpWholeBodyInterface: _dep_component_set = 1, _dep_component = dynamics
~~~
they are probably leftovers YCM debug prints.

**Comment by traversaro** [comment]
Any idea @drdanz ?

**Comment by drdanz** [comment]
I forgot some debug message in one of the latest commits.
Fixed in robotology/ycm@e9bfa3968c1141d8e5a416f648b0a862cf857e2e

**Comment by traversaro** [comment]
Thanks, great!
